hadoop-hdds/rocks-native/src/CMakeLists.txt

```
@@ -0,0 +1,78 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# CMake configuration.
+#
+
+cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
```

Does 2.8 work?

```suggestion
cmake_minimum_required(VERSION 2.8)
```

Since Ozone had another CMakeLists that uses the older version, we better be consistent: https://github.com/apache/ozone/blob/ac8aee7f8521ff3b70c8cc293432cee5d774896d/tools/fault-injection-service/CMakeLists.txt#L20

---

Also:

> The FATAL_ERROR option is accepted but ignored by CMake 2.6 and higher.

https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html